Automattic/wp-calypso
# Licensing Portal: implement the unassign licenses feature

#### Proposed Changes

* Licensing Portal: add the possibility to unassign a license from a site. Thus the users can unlinking a license from a specific site and reuse it on another site.

#### Testing Instructions

* Make sure you have a wordpress.com website.
* Open up Licensing -> Licenses.
* Issue a new license, if don't have one.
* Open some license details. If this license is not assigned to some site, please do it.
* The "Unassign License" button should appear on the bottom left corner.
* Click on the "Unassign License" button. An alert modal should appear with information about this action and ask for you to confirm that want to unassign that license from the site.

#### Screenshots
![image](https://user-images.githubusercontent.com/5550190/182459485-440f2cba-f413-4b78-805b-578d499e2e20.png)

![image](https://user-images.githubusercontent.com/5550190/182459596-5a93412e-a2c0-48aa-b1e3-ae2b241f43dd.png)


#### Pre-merge Checklist


Related to #
